src/dialogs/alwaysonbotDialogs/OASBenefit/applicationStatusDialog.ts

```diff
@@ -0,0 +1,43 @@
+import {
+    ComponentDialog, ConfirmPrompt, WaterfallDialog,PromptValidatorContext, ChoiceFactory,Choice,ListStyle,ChoicePrompt, WaterfallStepContext,DialogTurnResult
+} from 'botbuilder-dialogs';
+import { ContinueAndFeedbackDialog, CONTINUE_AND_FEEDBACK_DIALOG_STEP } from '../Common/continueAndFeedbackDialog';
+import i18n from '../../locales/i18nconfig';
+import { COMMON_CHOICE_CHECK_DIALOG,CommonChoiceCheckDialog } from '../UpdateProfile/UpdatePhoneNumber/commonChoiceCheckDialog';
+
+import { CommonPromptValidatorModel } from '../../../models/commonPromptValidatorModel';
+
+const WATERFALL_DIALOG = 'waterfallDialog';
```

Comment:
I  think naming them like this is a duplication. if you want to reuse them you can put this const WATERFALL_DIALOG to alwaysOnBotDialog.ts  then import to every single step. or you can give them a unique name like we did in callback/unblock dialogs